dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/test-applications/packaging/Packaging10EJB.ear/JPAAnoOverrideEJB.jar/META-INF/persistence.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd"
+    version="1.0">
+
+    <persistence-unit name="Cleanup" transaction-type="RESOURCE_LOCAL">
```

Comment:
Do these "Cleanup" persistence units get used anywhere in the application? I don't see any injections for them.

Reply:
nah they can go.